### PR TITLE
feat: add /api/chat e2e test (uses openai key)

### PR DESCRIPTION
**Pull request by thucpn**

### CODE REVIEW

**Review by marcusschiesser — APPROVED** [review]

**Review by marcusschiesser — APPROVED** [review]

File: packages/create-llama/e2e/utils.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ export function runCreateLlama(
     "--model",
     MODEL,
     "--open-ai-key",
-    "testKey",
+    process.env.OPENAI_API_KEY || "testKey",
```
Comment (marcusschiesser) [inline packages/create-llama/e2e/utils.ts:99]: @thucpn 

In the job, we're still getting `testKey` (https://github.com/run-llama/LlamaIndexTS/actions/runs/7246220748/job/19737708679) - might it be that `test` is starting a new process and this child process doesn't have the right `process.env`?

Comment (marcusschiesser) [reply packages/create-llama/e2e/utils.ts:99]: not the case as it works here: https://github.com/thucpn/LlamaIndexTS/actions/runs/7245927631/job/19736878992

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -36,6 +36,8 @@ jobs:
         working-directory: ./packages/create-llama
       - name: Run Playwright tests
         run: pnpm exec playwright test
+        env: 
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
```
Comment (marcusschiesser) [inline .github/workflows/e2e.yml:40]: @yisding please check if `OPENAI_API_KEY` is set correctly, as e2e tests are not running

Comment (yisding) [reply .github/workflows/e2e.yml:40]: It says it was set 4 days ago, but I tried adding it again (with a different key)

**Pull request closed without merging**